README.md

```diff
@@ -10,17 +10,20 @@
    kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
    ```
 
-2. Having the secrets to pull docker images in your cluster. When using the `us.icr.io` container registry, the secret obtained by running the following command is required is required.
+2. If you are not changing the code, you can use the sample public image `icr.io/gitsecure/sample-admission-controllers:0.0.2` for testing. If you are changing the code and wants to build a new image, you can follow these instructions:
```

Comment:
would be nice if we could stick it into https://github.com/orgs/OpenSecureSupplyChain/packages (ghcr.io/OpenSecureSupplyChain/sample-admission-controllers), just in case we need update it while your out (save you sharing your password with us :P )

Reply:
good idea..let's do that.

Reply:
We can get this set up tomorrow.

Reply:
are yeah I have seen this before

Try: 

`docker tag icr.io/gitsecure/sample-admission-controllers:0.0.2 ghcr.io/opensecuresupplychain/tkn-admission-controllers:0.0.2`

Reply:
github seems to case even everything.

Reply:
I don't seem to have permission:)
```
docker push ghcr.io/opensecuresupplychain/tkn-admission-controllers:0.0.2
The push refers to repository [ghcr.io/opensecuresupplychain/tkn-admission-controllers]
3225414cda27: Preparing
43656e43813a: Preparing
16679402dc20: Preparing
denied: permission_denied: create_package
```

Reply:
one sec, will take a look..

Reply:
should work now, try again pls.